test/language/statements/function/13.2-17-1.js

```diff
@@ -31,23 +32,12 @@ description: >
             var verifyValue = false;
             verifyValue = typeof fun.prototype.constructor === "function";
```

Comment:
Turn this into an `assert.sameValue`, please

Reply:
Fixed.